src/Aspose.Imaging.Cloud.Sdk/Model/Requests/ModifyGifRequest.cs

```diff
@@ -1,5 +1,5 @@
 // --------------------------------------------------------------------------------------------------------------------
-// <copyright company="Aspose" file="GetImageGifRequest.cs">
+// <copyright company="Aspose" file="ModifyGifRequest.cs">
 //   Copyright (c) 2019 Aspose Pty Ltd. All rights reserved.
 // </copyright>
```

Comment:
Please, use following copyright message pattern :
Copyright (c) <file creation year>-<current year> Aspose Pty Ltd. All rights reserved.

For example :
Copyright (c) 2018-2019 Aspose Pty Ltd. All rights reserved.

Reply:
@aifeigin done.